src/Sentry.NLog/SentryTarget.cs

```diff
@@ -196,6 +208,14 @@ protected override void InitializeTarget()
             }
         }
 
+        /// <inheritdoc />
+        protected override void FlushAsync(AsyncContinuation asyncContinuation)
+        {
+            _hubAccessor()
+                .FlushAsync(Options.FlushTimeout)
+                .ContinueWith(t => asyncContinuation(t.Exception));
```

Comment:
This looks good and I'd love to merge but it would be nice to cover this with tests somehow. Is it doable?

Reply:
Oh for sure I'll get on that real quick 🤦‍♂

Reply:
Okay, I added the test a few days ago, but it took me a while to get it to pass, which was because the flush timeout defaults were different in Sentry.NLog and NLog.

I had initially set the default `SentryNLogOptions.FlushTimeout` to match the default `SentryOptions.ShutdownTimeout` (2 sec), instead of the default `NLog` flush timeout of 15 seconds. I've updated that as well.

It may be worth documenting somewhere that if the sentry timeout is shorter than the NLog timeout, the Sentry flush it might not get hit.